### PR TITLE
WRN-11796: update qa-videoplayer sample

### PR DESCRIPTION
**Pull request by 0x64**
Enact-DCO-1.0-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Update qa-videoplayer sample to play selected loop repeatedly.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-11796

### CODE REVIEW

File: samples/qa-videoplayer/src/views/MainPanel.js [hunk 1]
```diff
@@ -1,30 +1,22 @@
...
 
 	static displayName = 'SelectableVideoPlayer';
 
-	static propTypes = {
-		css: PropTypes.object
-	};
-
 	state = {
-		selection: null
+		selection: []
```
Comment (MikyungKim) [inline samples/qa-videoplayer/src/views/MainPanel.js:12]: Let's add `selecting` initial state.

File: samples/qa-videoplayer/src/views/MainPanel.js [hunk 2]
```diff
@@ -59,29 +50,31 @@ const SelectableVideoPlayer = class extends Component {
 	handleSeekOutsideSelection = (ev) => {
```
Comment (MikyungKim) [inline samples/qa-videoplayer/src/views/MainPanel.js:51]: Remove the empty line.

File: samples/qa-videoplayer/src/views/MainPanel.js (hunk 1 above)
Comment (MikyungKim) [reply samples/qa-videoplayer/src/views/MainPanel.js:12]:
```suggestion
		section: []
```

File: samples/qa-videoplayer/src/views/MainPanel.js [hunk 3]
```diff
@@ -59,29 +50,31 @@ const SelectableVideoPlayer = class extends Component {
...
+			} else if (ev.time > selectionEnd) {
+				// this.video.pause();
+				this.video.seek(selectionEnd);
+			}
 
-		// or remove the selection and allow the default behavior
-		// this.setState({
-		// 	selection: null
-		// });
+			// or remove the selection and allow the default behavior
```
Comment (MikyungKim) [inline samples/qa-videoplayer/src/views/MainPanel.js:66]: Let's remove unused code.

File: samples/qa-videoplayer/src/views/MainPanel.js (hunk 1 above)
Comment (0x64) [reply samples/qa-videoplayer/src/views/MainPanel.js:12]: Removed `selecting` state.

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**